iceoryx_posh/include/iceoryx_posh/runtime/posh_runtime.hpp

```diff
@@ -76,9 +84,10 @@ class PoshRuntime
     /// @return cxx::expected<InstanceContainer,Error>
     /// InstanceContainer: on success, container that is filled with all matching instances
     /// Error: if any, encountered during the operation
-    /// Error::kPOSH__SERVICE_DISCOVERY_INSTANCE_CONTAINER_OVERFLOW : Number of instances can't fit in instanceContainer
-    /// Error::kIPC_INTERFACE__REG_UNABLE_TO_WRITE_TO_ROUDI_CHANNEL : Find Service Request could not be sent to RouDi
-    cxx::expected<InstanceContainer, Error> findService(const capro::ServiceDescription& serviceDescription) noexcept;
+    /// FindServiceError::INSTANCE_CONTAINER_OVERFLOW : Number of instances can't fit in instanceContainer
+    /// FindServiceError::UNABLE_TO_WRITE_TO_ROUDI_CHANNEL : Find Service Request could not be sent to RouDi
```

Comment:
I think the errors do not need to be listed here anymore since there is now an own enum only for this errors